src/screens/CreatePost/elements/UserProfile.js

```diff
@@ -4,10 +4,10 @@ import {Image, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 import AnonymousAvatar from '../../../components/AnonymousAvatar';
 import AnonymousProfile from '../../../assets/images/AnonymousProfile.png';
 import AnonymousUsername from '../../../components/AnonymousUsername';
-import {ToggleSwitchAnon} from '../../../hooks/useToggleColors';
+import ToggleSwitch from '../../../components/ToggleSwitch';
 import {POST_VERSION} from '../../../utils/constants';
+import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `ToggleSwitch` has been updated and the import statements for `colors` and `fonts` have been added. The old import statement for `ToggleSwitchAnon` and `COLORS` have been removed. This change is consistent with the PR description and there are no issues with these changes.
```diff
- import {ToggleSwitchAnon} from '../../../hooks/useToggleColors';
- import {COLORS} from '../../../utils/theme';
+ import ToggleSwitch from '../../../components/ToggleSwitch';
+ import {colors} from '../../../utils/colors';
+ import {fonts} from '../../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->